doc/user/content/releases/v0.33.md

```diff
@@ -20,7 +20,7 @@ CREATE CONNECTION kafka_connection TO KAFKA (
 );
 ```
 
-* Add [`mz_internal.mz_source_status`](/sql/system-catalog/mz_internal/#mz_source_status) and
+* Add [`mz_internal.mz_source_status`](/sql/system-catalog/mz_internal/#mz_source_statuses) and
```

Comment:
Is this bad? I'm not sure.

Reply:
I think this is a good approach! Keeping the original content for posterity, but update now dead links.

Reply:
Not unethical at all! 🙇‍♀️